modules/motions/BackgroundPlaCameraAction.cpp

```diff
@@ -132,25 +147,25 @@ void BackgroundPlaCameraAction::run()
   if(!isMetPreCondition()) return;
 
   // 撮影のため回頭
-  AngleRotation preRotation(robot, preTargetAngle, targetRotationSpeed, isClockwise);
+  PidGain prePidGain = { kp, ki, kd };
+  IMUAngleRotation preRotation(robot, preTargetAngle, basePower, isClockwise, prePidGain);
   preRotation.run();
 
-  // 動作安定のためのスリープ
-  this_thread::sleep_for(chrono::milliseconds(10));
+  // 綺麗な写真の撮影のためのスリープ
+  this_thread::sleep_for(chrono::milliseconds(100));
 
   PlaCameraAction plaCameraAction(robot, threshold, minArea, roi);
 
-  cv::Mat frame;
-
-  // 判定用のフレームの獲得
-  for(int i = 0; i < 5; ++i) {
-    robot.getCameraCaptureInstance().getFrame(frame);
-    this_thread::sleep_for(chrono::milliseconds(33));
-  }
-
   // もし初回で正面であればPlaCameraActionを実行、他の方向なら２回目でPlaCameraActionを実行、判定できなければ4回PlaCameraActionを実行
```

Comment:
処理を変えたから、コメントも修正した方がいいね
```suggestion
  // もし初回で正面であれば1回目のみPlaCameraActionを実行。他の方向なら２回目のみPlaCameraActionを実行。判定できなければ4回PlaCameraActionを実行すべきだが、本番の大会では時間の都合上、初回正面判定と同様、1回目のみPlaCameraActionを実行。
```